rosdep/base.yaml

```
@@ -2649,6 +2649,15 @@ libcairo2-dev:
   opensuse: [cairo-devel]
   rhel: [cairo-devel]
   ubuntu: [libcairo2-dev]
+libcamera:
+  debian:
+    sid: [libcamera-dev]
+  fedora:
+    "36": [libcamera]
+  gentoo: ['media-libs/libcamera']
+  nixos: [libcamera]
+  ubuntu:
+    impish: [libcamera-dev]
```

The package is also available on groovy and hirsute (and presumably the upcoming jammy).
```suggestion
    '*': [libcamera-dev]
    bionic: null
    focal: null
```

---

Thanks. I was actually looking for something like "impish and above". I didn't know you can add the asterisk (`*`) as a placeholder there and have the others explicitly deactivated.